### PR TITLE
Feature/output results to dir

### PR DESCRIPTION
**Pull request by antonioloison**
Option to output results to folder for each task of a project when doing `eval`
Results are saved to the desired folder path in a `.csv` file for each task

### CODE REVIEW

File: commands/common_args.py [hunk 1]
```diff
@@ -178,6 +178,12 @@ class TrainOptions:
             "Ex:  --additional-train-args-yolo " + json_string_yolo
         ),
     )
+    results_dir = click.option(
+        "--results-dir",
+        default=None,
+        type=str,
+        help=("Output directory name of the results."),
```
Comment (crsegerie) [inline commands/common_args.py:185]: I do not understand. Isn't this just like the verbose parameter ?

File: commands/common_args.py [hunk 2]
```diff
@@ -194,6 +200,12 @@ class EvaluateOptions:
         type=str,
         help=("Use a model trained on a different project to predict on project_id."),
     )
+    results_dir = click.option(
```
Comment (crsegerie) [inline commands/common_args.py:203]: repetition

File: commands/common_args.py (hunk 1 above)
Comment (crsegerie) [reply commands/common_args.py:185]: Output directory name of the evaluation results.

File: kiliautoml/utils/helpers.py [hunk 3]
```diff
@@ -334,7 +335,26 @@ def get_keys(my_dict):
...
+    return headers, table
+
+
+def print_and_save_evaluation(
+    job_name: JobNameT, evaluation: EvalResultsT, results_folder: Optional[str] = None
+):
+    headers, table = create_table(job_name, evaluation)
+    print(tabulate(table, headers=headers))
+
+    if results_folder is not None:
```
Comment (crsegerie) [inline kiliautoml/utils/helpers.py:348]: else ?

Comment (antonioloison) [reply kiliautoml/utils/helpers.py:348]: We already printed the evaluation table so there is no need for anything else here.

**Review by crsegerie — APPROVED** [review]

**Pull request merged**